tests/mocks/repository/lib/mock_kvset.c

```diff
@@ -219,7 +219,7 @@ _kvset_open(struct cn_tree *tree, uint64_t kvsetid, struct kvset_meta *km, struc
     int                i = 0, j;
 
     /* +1 for the singular hblock */
-    alloc_sz = sizeof(*mk) + (sizeof(u64) * (1 + km->km_kblk_list.n_blks + km->km_vblk_list.n_blks));
+    alloc_sz = sizeof(*mk) + (sizeof(u64) * (1 + km->km_kblk_list.idc + km->km_vblk_list.idc));
```

Comment:
u64

Reply:
We should sweep the entire code base and make that change

Reply:
Heh, yeah, but I'm not reviewing it!